### PR TITLE
[BIA-1061] - Creates K12StudentsDim on CEDS collection. 

### PR DESCRIPTION
**Pull request by jegasape**

### CODE REVIEW

File: CEDS/MSSQL/0005-View-K12StudentsDim-Create.sql [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        )
+BEGIN
+    DROP VIEW xref.K12StudentsDim;
+END;
+GO
+
+CREATE OR ALTER VIEW xref.K12StudentsDim AS
+(
+    SELECT 
+        CONCAT(Student.StudentUniqueId, '-', StudentSchoolAssociation.SchoolId) AS StudentKey,
```
Comment (jleiva-gap) [inline CEDS/MSSQL/0005-View-K12StudentsDim-Create.sql:19]: Update the name to StudentSchoolKey.

File: CEDS/MSSQL/0005-View-K12StudentsDim-Create.sql [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        '' AS RecordEndDateTime
+    FROM 
+        edfi.Student
+    LEFT JOIN 
+        edfi.StudentSchoolAssociation
+    ON 
+        Student.StudentUSI = StudentSchoolAssociation.StudentUSI
+    WHERE 
+        StudentSchoolAssociation.PrimarySchool = 1
+    GROUP BY
```
Comment (jleiva-gap) [inline CEDS/MSSQL/0005-View-K12StudentsDim-Create.sql:36]: It's not working. It requires StudentSchoolAssociation.SchoolId

File: CEDS/MSSQL/0005-View-K12StudentsDim-Create.sql [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+CREATE OR ALTER VIEW xref.K12StudentsDim AS
+(
+    SELECT 
+        CONCAT(Student.StudentUniqueId, '-', StudentSchoolAssociation.SchoolId) AS StudentKey,
+        Student.BirthDate,
+        StudentSchoolAssociation.ClassOfSchoolYear AS Cohort,
+        Student.FirstName,
+        Student.LastSurname,
+        Student.MiddleName,
```
Comment (jleiva-gap) [inline CEDS/MSSQL/0005-View-K12StudentsDim-Create.sql:24]: Add a Coalesce to return '' if null. MiddleName could be null.

File: CEDS/MSSQL/0005-View-K12StudentsDim-Create.sql [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+            AND TABLE_NAME = 'K12StudentsDim'
+        )
+BEGIN
+    DROP VIEW xref.K12StudentsDim;
+END;
+GO
+
+CREATE OR ALTER VIEW xref.K12StudentsDim AS
+(
+    SELECT
```
Comment (jleiva-gap) [inline CEDS/MSSQL/0005-View-K12StudentsDim-Create.sql:18]: Add a new key column
`CONCAT(Student.StudentUniqueId, '-', StudentSchoolAssociation.SchoolId, '-', StudentSchoolAssociation.ClassOfSchoolYear) as K12StudentsKey`

**Review by jleiva-gap — CHANGES_REQUESTED** [review]

File: CEDS/MSSQL/0005-View-K12StudentsDim-Create.sql (hunk 1 above)
Comment (jegasape) [reply CEDS/MSSQL/0005-View-K12StudentsDim-Create.sql]: Done.

**Review by jleiva-gap — APPROVED** [review]

**Pull request merged**